### PR TITLE
feat(tweaks): Cleanup a few things

### PR DESCRIPTION
**Pull request by zflamig**

### CODE REVIEW

File: src/components/layout/NavBar.jsx [hunk 1]
```diff
@@ -52,19 +52,19 @@ const NavLogo = styled.div`
 // TODO: due to issue https://github.com/styled-components/styled-components/issues/439,
 // bgcolor prop triggers react warning now, need to fix
 const NavItem = styled(Link)`
-  border-left: 1px solid #9b9b9b;
+  border-left: 2px solid #f4f4f4;
```
Comment (thanh-nguyen-dang) [inline src/components/layout/NavBar.jsx]: why did the color change on the border? think any change of design need to be discussed with @uchicagovivi @zflamig

Comment (uchicagovivi) [reply src/components/layout/NavBar.jsx]: sorry... my bad...
That color is #D1D1D1, and please name it “smoke"
I'm updating it on style guide too. thanks!

File: src/components/layout/TopBar.jsx [hunk 2]
```diff
@@ -20,27 +20,31 @@ const TopRight = styled.nav`
 
 // TODO: due to issue https://github.com/styled-components/styled-components/issues/439,
 const TopItem = styled(Link)`
-  border-right: 1px solid #9b9b9b;
+  border-right: 2px solid #6896c6;
```
[the remark this answers is not in the document]
Comment (thanh-nguyen-dang) [reply src/components/layout/TopBar.jsx]: how about this @zflamig @uchicagovivi ?

[the remark this answers is not in the document]
Comment (uchicagovivi) [reply src/components/layout/TopBar.jsx]: that looks awesome, thank you!!

**Pull request merged**